Remove unused makeCellId import from records.ts (lint warning)

## Description
npm run lint reports an unused import warning in records store:
src/stores/table-data/records.ts:23:10 warning 'makeCellId' is defined but never used @typescript-eslint/no-unused-vars

### Affects users
None

<!-- If not "None", how does this debt affect users? -->

### Affects developers
<!-- Give a rating. Delete all but one -->
Low
Lint warning appears in CI and local development. Unused import bloats bundle slightly and violates code hygiene standards.

<!-- If not "None", how does this debt affect developers? -->

## Suggested change
<!-- What is the proposed change to the codebase. -->
Remove unused import from line 23:

```
diff
src/stores/table-data/records.ts
- import { makeCellId } from './utils';
```

## Additional context
<!-- Add any other context about the problem or screenshots here. -->
Verify it with
npm run lint

--- a/mathesar_ui/src/stores/table-data/records.ts
+++ b/mathesar_ui/src/stores/table-data/records.ts
@@ -20,7 +20,7 @@ import type {
   RecordsSearchParams,
   ResultValue,
 } from '@mathesar/api/rpc/records';
-import { makeCellId, parseCellId } from '@mathesar/components/sheet/cellIds';
+import { parseCellId } from '@mathesar/components/sheet/cellIds';
 import type { Database } from '@mathesar/models/Database';
 import type { Table } from '@mathesar/models/Table';
 import {

